src/main/kotlin/org/rucca/cheese/task/service/TaskMembershipViewService.kt

```
@@ -0,0 +1,472 @@
+                .map { it.memberId }
+                .distinct()
+        val teamMemberUserDtoMap =
+            if (allTeamMemberUserIds.isNotEmpty()) userService.getUserDtos(allTeamMemberUserIds)
+            else emptyMap()
+
+        // 4. Fetch team owner IDs
+        val teamOwnerIdMap =
+            teamMemberIds.associateWith { teamId ->
+                try {
+                    teamService.getTeamOwner(teamId)
+                } catch (e: NotFoundError) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)